bin/cli.js

```diff
@@ -62,32 +62,30 @@ const parseFile = file => {
     case 'yaml':
       return yaml.parse(data);
     default:
-      return {};
+      throw new Error('Unsupported file format');
   }
 };
 
 const loadFiles = files => {
   const result = [];
-  files
-    .map(file => {
-      if (fs.existsSync(file + '.js')) {
-        return file + '.js';
-      } else if (fs.existsSync(file)) {
-        return file;
-      } else {
-        console.error('File does not exist:', file);
-        process.exit(1);
-        return '';
-      }
-    })
-    .forEach(file => {
-      if (fs.statSync(file).isDirectory()) {
-        const subfiles = fs.readdirSync(file).map(f => path.join(file, f));
-        result.push(...loadFiles(subfiles));
-      } else if (common.fileExt(file) === 'js') {
-        result.push(file);
-      }
-    });
+  // eslint-disable-next-line consistent-return
```

Comment:
Oh, so that line is indeed needed, I think putting it back better than disabling a rule.